grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesReader.groovy

```diff
@@ -32,18 +31,54 @@ import javax.jcr.Node as JcrNode
 @CompileStatic
 @SuppressWarnings("GrMethodMayBeStatic")
 class JcrNodesReader implements ItemReader<JcrNode> {
-
-    @Override
+	
+	private String excludePaths
+	private String[] excludePathList
+	private static final String[] EMPTY_ARRAY = new String[0];
+	
+	public void setExcludePaths(String excludePaths) 
+	{
+        this.excludePaths = excludePaths
+        if(!this.excludePaths.isEmpty() && this.excludePaths != null)
+            this.excludePathList = excludePaths.split("\\|");
+		else
+            this.excludePathList = EMPTY_ARRAY;
+	}
+	
+	@Override
     JcrNode read() throws Exception, UnexpectedInputException, ParseException, NonTransientResourceException {
         Iterator<JcrNode> nodeIterator = theNodeIterator()
         if(nodeIterator == null) throw new IllegalStateException("nodeIterator must be set.")
-
-        if(nodeIterator.hasNext()) {
-            nodeIterator.next()
+        JcrNode returnNode = null;
+        while(true)
+        {
```

Comment:
Minor, but need to fix formatting to be consistent with the rest of the project.